src/esnext.coffee

```diff
@@ -0,0 +1,157 @@
+###
+Cache for source code transpiled by 6to5.
+
+Inspired by https://github.com/atom/atom/blob/6b963a562f8d495fbebe6abdbafbc7caf705f2c3/src/coffee-cache.coffee.
+###
+
+crypto = require 'crypto'
+fs = require 'fs-plus'
+path = require 'path';
+to5 = require '6to5-core'
+
+stats =
+  hits: 0
+  misses: 0
+
+defaultOptions =
+  # The Chrome dev tools will show the original version of the file
+  # when the source map is inlined.
+  'sourceMap': 'inline'
```

Comment:
We typically drop the quotes around the keys unless one of the keys needs it (such as because if contains a `-` or something).  Since none of these keys appear to need it, I'd say drop all the quotes around them in the `defaultOptions` object.